advocacy_docs/partner_docs/Cyberark/05-Usingcyberark.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured your EDB Postgres Advanced Server and CyberArk, as stated in the Configuring section, you will be able to then use CyberArk (CPM Central Policy Manager) functionality. 
+
+Authorized users can change passwords that are stored in the Safe through the PVWA (Password Vault Web Access). These passwords can be changed manually or replaced by a new password that is randomly generated by the Central Policy Manager. The CPM generates unique and highly secure passwords using the password policy and the random password generation mechanism. 
```

Comment:
Would add (CPM) after the first mention of Central Policy Manager so when we say CPM throughout the guide we know what it is for.

Reply:
done